src/routes/search.rs

```diff
@@ -44,6 +61,11 @@ pub fn search() -> Result<Json<Vec<Manifest>>, Error> {
         file.read_to_string(&mut s)?;
         manifests.push(toml::from_str(&s)?);
     }
+
+    let sorter: fn(&Manifest, &Manifest) -> Ordering;
+    sorter = |a, b| b.metadata().created_at().cmp(&a.metadata().created_at()); // Default sort by create_at
+    manifests.sort_by(sorter);
```

Comment:
Do you really need to do this in two steps ?
This doesn't work?

```rust
manifests.sort_by(|a, b| {
    b.metadata().created_at().cmp(&a.metadata().created_at()
});

// or, if you have problem with type inference

manifests.sort_by(|a: &Manifest, b: &Manifest| {
    b.metadata().created_at().cmp(&a.metadata().created_at()
});

```

Even if you don't, you probably don't need to first declare `sorter` and then assign it, you can probably do both in one line.

(Be aware that this is an ascii sort on date, which is different than a date sort. Remember to ensure that the sort is still good when you'll change the created date's type).